core/docs/build.md

```diff
@@ -41,6 +41,12 @@ Install the required packages, depending on your operating system.
   sudo pacman -S scons sdl2 sdl2_image
   ```
 
+* __NixOS__:
+
+  ```sh
+  nix-shell -p gnumake python3 scons SDL2_image SDL2.dev pkg-config
```

Comment:
There is `shell.nix` file in the root of the project. I usually run `nix-shell` in the root (which picks up `shell.nix`) and then `cd core` to do core stuff.

Reply:
That's probably the right way to do it for the developement. I will rebase.